tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "strictNullChecks": false,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true,
```

Comment:
These two are implied with `strict`

Reply:
Fixed in https://github.com/atom/atom-select-list/pull/28/commits/8818ab07a04e7b773560c543f634f5c9672fed9d